Sluggy/Sluggy.cs

```diff
@@ -70,10 +72,11 @@ public static string ToSlug(this string text, string separator, ITranslationStra
             }
 
             return text
-                .Split()
-                .Where(t => t.Length != 0)
-                .Select(t => strategy.Translate(t))
-                .Join(separator);
+                 .Split()
+                 .Where(t => t.Length != 0)
+                 .Select(t => strategy.Translate(t))
+                 .Where(t => t.Length != 0)
```

Comment:
It seems strange to have this 2nd check. Shouldn't the translate strategies already ensure this step isn't needed?

Reply:
Or maybe remove the first Where.

Reply:
yeah i know.
The first where is to avoid the strategies to process empty strings.
The second is to cleanup any empty strings.